core/chains/evm/txmgr/export_orm_test.go

```diff
@@ -0,0 +1,16 @@
+package txmgr
+
+// A way to export and test private methods and variables in the txmgr orm
+type Eorm struct {
+	*orm
+}
+
+func NewEorm(o ORM) *Eorm {
+	return &Eorm{o.(*orm)}
+}
+
+var UpdateEthTxAttemptUnbroadcast = updateEthTxAttemptUnbroadcast
```

Comment:
I've begun to test some private functions here (note they aren't methods of orm struct...they're just placed in orm.go)

Reply:
Are we stuck writing tests for these in `package txmgr_test`? It would be cleaner to test in `package txmgr` if possible.

Reply:
nit/ we tend to name these kind of files `helpers_test.go`

Reply:
> Are we stuck writing tests for these in `package txmgr_test`?

We aren't stuck writing tests for these exported/publicly wrapped functions in the `txmgr_test` package. Note the package for this `export_orm_test.go` file is the `txmgr` package so we can use this anywhere so long as we import the `txmgr` package.

The way I test these functions is I import them into the `txmgr_test` package like so https://github.com/smartcontractkit/chainlink/pull/8523/files#diff-f9ab5defa815e4a7577a4cdceca90e5df113f46a3bb79badd6027095bd162745R697.

> It would be cleaner to test in `package txmgr` if possible.

The orm_test.go is declared in the `txmgr_test` package so I'm testing the exported functions there (declaring them in `txmgr` package in this `export_orm_test.go` file though so I have access to the orm struct) since I'm just following our testing convention.

Reply:
> nit/ we tend to name these kind of files `helpers_test.go`

Will do 👍